src/components/Agents/Add/AddAgentForm.tsx

```diff
@@ -169,6 +174,27 @@ export default function AgentForm({
                         hint="How often to perform a full reconciliation of changes (in addition to real-time changes from Kubernetes events), set higher for larger clusters."
                       />
                     )}
+
+                    <FormikSwitchField
+                      options={[
+                        { label: "Enabled", key: true },
+                        { label: "Disabled", key: false }
+                      ]}
+                      name="pushTelemetry.enabled"
+                      label={
+                        <div className="flex w-full flex-col">
+                          <span>Telemetry</span>
+                        </div>
+                      }
+                    />
+                    {Boolean(values.pushTelemetry?.enabled) === true && (
+                      <FormikTextInput
+                        name="pushTelemetry.topologyName"
+                        label="Topology Name"
```

Comment:
Default this to the agent name
```suggestion
                        label="Label"
```